### DISCUSSION TITLE
Error on fresh install

### DISCUSSION THREAD

**Discussion by onjaysantana** [givanz/Vvveb#188] [Q&A]
I just installed vvveb on the root of a subdomain. When I first went to the page, I got a "we're having trouble finding that site" browser error. Then when I pointed to the install subfolder, the installer came up and installation went normally.

I can log into the admin section properly. The pages that were created on installation all show a 404 not found error when I click on them.
On the other hand, if I go to the site (not admin subfolder), the post I created (while in admin) comes up as the main page. 

Going to the posts section, all the posts created on install and my new one are all listed and showing their related images. Again, if I click "view" on any on them, I get a 404 not found page. HOWEVER, I can still edit and save the posts. if I click (edit) in the posts list.

BUT, if I am in the post and click the option to edit in the editor, the editor opens, but the the page that comes up to edit is the vvveb 404 not found page.

If I point the browser to the "public" subfolder, the vvveb page comes up, but with no formatting--like a html page full of links. Clicking on any of the links, opens the vvveb page properly, but with all the problems listed above. Mainly that none of the pages can be found when trying to "view" them. I can still open all the pages, edit them, and save them.

I hope this is enough to know what I need to do to fix this. It seems I remember this problem mentioned somewhere in the documentation, but I can't find it again. 

I sure would appreciate some help with this. Thanks.
JS

**Comment by givanz** [comment]
Usually 404 not found error for posts/pages are caused by missing or inactive mod rewrite apache module.

Can you please check in admin > tools > system info https://demo.vvveb.com/admin/index.php?module=tools%2Fsysteminfo&phpinfo=true in the apache section if `mod_rewrite` module is present?

![rewrite](https://github.com/user-attachments/assets/e4083bac-8c07-41ec-8210-05d536117912)

If it's not present then you need to enable the module from your cpanel.

If your hosting uses nginx instead of apache then you need to edit the nginx configuration and add

```nginx
  location / {
        try_files $uri $uri/ /index.php?$query_string;
  }
```

You can check the included nginx.conf https://github.com/givanz/Vvveb/blob/master/nginx-live-nocache.conf for more configuration options.
